scripts/e2e/cmd/runner/helper.go

```diff
@@ -503,6 +510,129 @@ func applyYaml(clientset *clientset.Clientset, crdClient *versioned.Clientset, n
 	return nil
 }
 
+func deleteYaml(clientset *clientset.Clientset, crdClient *versioned.Clientset, namespaceName string, fileName string) error {
+	// create objects in the yaml
+	fileObjects, err := parseK8sYaml(fileName)
+	if err != nil {
+		return err
+	}
+
+	for _, objs := range fileObjects {
```

Comment:
It would be nice if it were possible to refactor `deleteYaml`, `applyYaml`, and `updateYaml` as wrappers around a single common loop but I can't think of a nice way to do it